### PR TITLE
Print factored count over candidates count

### PR DESCRIPTION
**Pull request by daedalus**

### CODE REVIEW

File: python/FACTOR.py [hunk 1]
```diff
@@ -632,6 +634,7 @@ def mine(self, mine_latest_block = True, coinbase_message = "", scriptPubKey = N
                     #Check if the current block race has been won already
                     if rpc_getblockcount() + 1 != block.blocktemplate["height"]:
                         print("[LOST] Total lost time:", time() - self.START, " Seconds." )
+                        print("Factored: %d of %d" % (self.Count, len(candidates))
```
Comment (ReverseControl) [inline python/FACTOR.py:637]: Missing a parenthesis.

**Review by ReverseControl — CHANGES_REQUESTED** [review]
Looks good. Just fix the missing parenthesis of the print statement. You might also want to print that stat above the "Total lost time" line and not below.

File: python/FACTOR.py (hunk 1 above)
Comment (daedalus) [reply python/FACTOR.py]: Fixed

**Comment by daedalus** [comment]
Plase take a look at the latest commit.

File: python/FACTOR.py [hunk 2]
```diff
@@ -535,7 +535,7 @@ def compute_raw_hash(self):
     #TODO: replace and raise an error if no scriptPubKey is given for master branch.
     def mine(self, mine_latest_block = True, coinbase_message = "", scriptPubKey = None ):
         global primorial_base, siever, SIEVE_MAX_LEVEL
-        self.Count = 0
+
```
Comment (ReverseControl) [inline python/FACTOR.py:538]: You need to leave this here. This count should be initialized to zero every time we enter this function. If it is not globally defined line 620 will throw an error, and even if it is the printed value performed in line 620 will be wrong. So, initialize to zero here.

Comment (ReverseControl) [reply python/FACTOR.py:538]: @daedalus

**Pull request closed without merging**